src/browser/pages/vscode.ts

```diff
@@ -24,13 +24,13 @@ type NlsConfiguration = {
  * Helper function to create the path to the bundle
  * for getNlsConfiguration.
  */
-export function createBundlePath(_resolvedLanguagePackCoreLocation: string, bundle: string) {
+export function createBundlePath(_resolvedLanguagePackCoreLocation: string | undefined, bundle: string) {
   // NOTE@jsjoeio - this comment was here before me
   // Refers to operating systems that use a different path separator.
   // Probably just Windows but we're not sure if "/" breaks on Windows
   // so we'll leave it alone for now.
   // FIXME: Only works if path separators are /.
-  return _resolvedLanguagePackCoreLocation + "/" + bundle.replace(/\//g, "!") + ".nls.json"
+  return (_resolvedLanguagePackCoreLocation || "") + "/" + bundle.replace(/\//g, "!") + ".nls.json"
```

Comment:
🧹 Originally, we were using `_resolvedLanguagePackCoreLocation` or passing in an empty string. We moved the logic from outside the function into the function since the caller shouldn't have to change the argument functions if the value is `undefined`. Instead, the function should handle this.